flutter_vlc_player_platform_interface/lib/src/enums/subtitles/vlc_subtitle_text_direction.dart

```diff
@@ -0,0 +1,14 @@
+enum VlcSubtitleTextDirection {
+  LTR,
+  RTL,
+  AUTO,
+}
+
+extension VlcSubtitleTextDirectionExtensionMap on VlcSubtitleTextDirection {
+  static const valueMap = {
+    VlcSubtitleTextDirection.LTR: 0,
+    VlcSubtitleTextDirection.RTL: 1,
+    VlcSubtitleTextDirection.AUTO: 2,
+  };
+  int get value => valueMap[this];
```

Comment:
Let's use Effective Dart naming convention for enums: https://dart-lang.github.io/linter/lints/constant_identifier_names.html
```suggestion
enum VlcSubtitleTextDirection {
  ltr,
  rtl,
  auto,
}

extension VlcSubtitleTextDirectionExtensionMap on VlcSubtitleTextDirection {
  static const valueMap = {
    VlcSubtitleTextDirection.ltr: 0,
    VlcSubtitleTextDirection.rtl: 1,
    VlcSubtitleTextDirection.auto: 2,
  };
  int get value => valueMap[this];
```